stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/usecase/GetChannelImpl.kt

```diff
@@ -0,0 +1,35 @@
+package io.getstream.chat.android.livedata.usecase
+
+import io.getstream.chat.android.client.utils.Result
+import io.getstream.chat.android.livedata.ChatDomainImpl
+import io.getstream.chat.android.livedata.controller.ChannelController
+import io.getstream.chat.android.livedata.utils.Call2
+import io.getstream.chat.android.livedata.utils.CallImpl2
+import io.getstream.chat.android.livedata.utils.validateCid
+
+public interface GetChannel {
```

Comment:
A bit strange that we have GetThread, CetChannel use cases returning controllers. Would be better to name them GetThreadController, CetChannelController, etc. Unfortunately, these are breaking changes.

Reply:
Get Channel is new, I like the GetChannelController name